msteams-platform/m365-apps/specify-runtime-requirements.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Specify App Runtime Requirements
+description: Specify app runtime requirements, such as one-way or mutual dependencies and capabilities, to tailor user experience in different Microsoft 365 hosts.
+ms.date: 10/14/2024
+ms.author: mosdevdocs
+author: erikadoyle
+ms.topic: conceptual
+ms.localizationpriority: medium
+ms.subservice: m365apps
+---
+# Specify Microsoft 365 host runtime requirements in app manifest
+
+> [!NOTE]
+>
+> The ability to specify Microsoft 365 host runtime requirements in app manifest (previously called Teams app manifest) is in [public developer preview](../resources/schema/manifest-schema-dev-preview.md).
+
+When you upgrade your Microsoft Teams personal tab or message extension app to use app manifest version 1.13 or later, it's available in other Microsoft 365 application hosts by default. However, if your app also includes capabilities not yet supported in certain hosts, your app might only partially load, resulting in unplanned user experiences.
+
+For example, consider an app that is defined with app manifest schema version 1.17 and includes a bot and a configuration tab that represents the bot's settings. The app would load in Outlook and Microsoft 365 (Office) app, but only surface the bot configuration tab to the user without the bot itself.
+
+To ensure high-quality app experiences and reach a wider user base, specify your app's runtime requirements in app manifest. This allows you to tailor its behavior in applicable Microsoft 365 hosts, or omit it from surfacing in contexts you're not ready to support.
```

Comment:
We should not state 'reach a wider user base' as that is contrarian to what the feature does - restrict the app surfacing in hubs where certain features are not supported. Please remove this.

Reply:
Thanks, good catch! Removed and reworded.